bpy_speckle/connector/operations/load_operation.py

```diff
@@ -46,8 +52,32 @@ def load_operation(context: Context) -> None:
 
     version_data = operations.receive(obj_id, transport)
 
+    # Create material mapping first
     material_mapping = render_material_proxy_to_native(version_data)
-    print(f"Created material mapping for {len(material_mapping)} objects")
+
+    definition_collections, definition_objects = instance_definition_proxy_to_native(
+        version_data, material_mapping
+    )
+
+    definitions_root_collection = None
+    if definition_collections:
+        definitions_root_collection = bpy.data.collections.new("InstanceDefinitions")
+
+        for collection in definition_collections.values():
+            definitions_root_collection.children.link(collection)
+
+    definition_object_ids = set()
+    for definition in find_instance_definitions(version_data).values():
+        if hasattr(definition, "objects"):
```

Comment:
Do we not have a class for InstanceDefinitions?

Would be simpler to loop through, cast to that type, and now you know that `objects` property exists, and is definitely a list?